components/test_raftstore/src/common-test.toml

```diff
@@ -116,8 +116,3 @@ wake-up-delay-duration = "100ms"
 enable = false
 advance-ts-interval = "10ms"
 scan-lock-pool-size = 1
-[raftstore.store-batch-system]
```

Comment:
Why remove these?

Reply:
@5kbpers This is an early wrong serialization, it should be kept as `[raftstore] store-pool-size`